CHANGELOG.md

```diff
@@ -33,9 +39,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [v0.8.1][v0.8.1_log] - 2020-02-07
 ---------------------
 
-### Added
```

Comment:
Why is this part removed in this PR?

Reply:
Fixed. Note, to be consistent with changelog for earlier releases, I have moved the `Added` section before `Changed` section.